trantor/net/inner/TimerQueue.cc

```diff
@@ -270,12 +270,27 @@ void TimerQueue::reset(const std::vector<TimerPtr> &expired,
     loop_->assertInLoopThread();
     for (auto const &timerPtr : expired)
     {
-        if (timerPtr->isRepeat() &&
-            timerIdSet_.find(timerPtr->id()) != timerIdSet_.end())
+        // if (timerPtr->isRepeat() &&
+        //     timerIdSet_.find(timerPtr->id()) != timerIdSet_.end())
+        // {
+        //     timerPtr->restart(now);
+        //     insert(timerPtr);
+        // }
```

Comment:
Please delete useless lines and don't worry, because we have git version control.